core/src/main/java/com/sequenceiq/cloudbreak/service/user/RemoteUserDetailsService.java

```diff
@@ -124,6 +144,64 @@ public void evictUserDetails(String updatedUserId, String filterValue) {
         LOGGER.info("Remove userid: {} / username: {} from user cache", updatedUserId, filterValue);
     }
 
+    @Override
+    public void deleteUser(CbUser admin, String userId) {
+        CbUser userToDelete = getDetails(userId, UserFilterField.USERID);
+        String accessDenidedMessage = null;
+        LOGGER.info(String.format("%s / %s tries to delete %s", admin.getUserId(), admin.getUsername(), userId));
```

Comment:
String.format not required here; the slf4 logger can be passed arguments like this:
LOGGER.info("{} / {} triese to delete {}", arg1, arg2, arg3)